src/mesh/mesh_dual_weighted_residual_error.cpp

```diff
@@ -0,0 +1,388 @@
+#include <vector>
```

Comment:
It looks like this file is defining a lot of functions for another .h file that isn't consistent with naming.

Reply:
Names are now consistent in the new commit.